src/components/blocks/FilterOptions.tsx

```diff
@@ -10,7 +11,7 @@ import { TCorpusTypeDictionary, TThemeConfig, TThemeConfigFilter } from "@/types
 import { TextInput } from "@/components/forms/TextInput";
 
 const getTaxonomyAllowedValues = (corporaKey: string, taxonomyKey: string, corpus_types: TCorpusTypeDictionary) => {
-  const allowedValues = corpus_types[corporaKey].taxonomy[taxonomyKey]?.allowed_values;
+  const allowedValues = get(corpus_types[corporaKey].taxonomy, taxonomyKey)?.allowed_values || [];
```

Comment:
pretty certain we can do this with standard JS with something like
```ts
(corpus_types?.[corporaKey]?.taxonomy ?? taxonomyKey)?.allowed_values ?? []
```

Reply:
Actually I've got the logic above wrong, so maybe ignore this

Reply:
Yeh, the issue is with nested taxonomies, for example for "types" I want the getter to find the key _within_ the `_documents` object.